spec/spec.opts

```diff
@@ -1,6 +1,5 @@
 --format
 documentation
 --colour
---loadby
```

Comment:
sorry, this was committed by accident, I will purge it again. I had to remove this line to use parallel_spec for local testing.